mgmt_cli_test.py

```diff
@@ -695,16 +695,16 @@ def test_client_encryption(self):
         healthcheck_task = mgr_cluster.get_healthcheck_task()
 
         self.db_cluster.enable_client_encrypt()
-        time.sleep(30)  # Make sure healthcheck task is triggered
-
-        # Scylla-manager should pick up client encryption setting automatically
-        healthcheck_task.wait_for_status(list_status=[TaskStatus.DONE], step=5, timeout=240)
 
+        # SM caches scylla nodes configuration and the healthcheck svc is independent from the cache updates.
+        # Cache is being updated periodically, every 1 minute following the manager config for SCT.
+        # We need to wait until SM is aware about the configuration change.
         mgr_cluster.update(
             client_encrypt=True,
             force_non_ssl_session_port=mgr_cluster.sctool.is_minimum_3_2_6_or_snapshot
         )
-        time.sleep(30)  # Make sure healthcheck task is triggered
+        time.sleep(240)
+
         healthcheck_task.wait_for_status(list_status=[TaskStatus.DONE], step=5, timeout=240)
```

Comment:
Just for my understanding, why can't we just adjust the timeout here in ` healthcheck_task.wait_for_status...` instead of adding sleep?

Reply:
As per my understanding, "wait_for_status" is just querying the specified task from manager and waits until expected status (DONE/RUNNING/ERROR etc) appears.
Here I need a full sleep for the test to wait until background service in manager (that is running ever 1 minute according to new configuration updated in this PR) read the update node config.

Reply:
Have you experimented with smaller timeouts? As I got it, the update takes place every 1 minute but the sleep here is 4 minutes. Do we need to wait for so long?

Reply:
I haven't experiment with lower timeouts :)
But I'd prefer to not block this PR until the exact expected sleep is found here.
I've chosen the value that is just safe.

Reply:
agree, it will take time, just for my understanding